pkg/kubelet/dockertools/manager.go

```diff
@@ -580,6 +581,16 @@ func (dm *DockerManager) runContainer(
 		// of CPU shares.
 		cpuShares = milliCPUToShares(cpuRequest.MilliValue())
 	}
+	var devices []dockercontainer.DeviceMapping
+	if nvidiaGPULimit.Value() != 0 {
+		// Experimental. For now, we hardcode /dev/nvidia0 no matter what the user asks for
+		// (we only support one device per node).
+		devices = []dockercontainer.DeviceMapping{
+			{"/dev/nvidia0", "/dev/nvidia0", "mrw"},
```

Comment:
As a datapoint, I manually verified that a gpu bindmount in rkt worked (prior to capabilities working though; 1.3 or 1.4 iirc). That implies to me that gpu bindmounting should work if we grant sufficient capabilities. Things that might matter: it should be mode `crw-rw-----` or thereabouts (which bindmounting accomplishes correctly for me on recent rkt 1.5). We might have to do some fiddling with `cgroup/devices/devices.{allow,deny}` as well, but I think we currently don't touch either so that too should just work.